docs/api/selector.md

```diff
@@ -13,7 +13,9 @@ follows:
 - class syntax (`.foo`, `.foo-bar`, etc.)
 - element syntax (`input`, `div`, `span`, etc.)
 - id syntax (`#foo`, `#foo-bar`, etc.)
-- attribute syntax (`[href="foo"]`, `[type="text"]`, and the other attribute selectors listed [here](https://developer.mozilla.org/en-US/docs/Learn/CSS/Introduction_to_CSS/Attribute_selectors).)
+- attribute syntax (`[href="foo"]`, `[type="text"]`, and the other attribute
+- universal syntax (`*`)
+selectors listed [here](https://developer.mozilla.org/en-US/docs/Learn/CSS/Introduction_to_CSS/Attribute_selectors).)
```

Comment:
This line needs to be moved back to "attribute syntax". It currently renders
>- attribute syntax ([href="foo"], [type="text"], and the other attribute
>- universal syntax (*) selectors listed here.)

Reply:
Oops, thanks!

Reply:
This hasn't been updated yet.

Reply:
fixed in eee9542